cloud-watch-to-slack-testing-58006136-c30b-43d1-880c-7b36f860e7c8/deployment/index.js

```diff
@@ -219,6 +219,13 @@ function dockstoreDeployerMessageText(message) {
   return message.message;
 }
 
+function s3ActivityMessageText(message) {
+  const userName = message.detail.userIdentity["userName"];
+  const eventName = message.detail["eventName"];
+  const bucketName = message.detail.requestParameters["bucketName"];
+  return `${userName} generated S3 event ${eventName} for bucket ${bucketName} in Dockstore ${dockstoreEnvironment} in region ${message.region}`;
```

Comment:
Is `message.region` the region of originating bucket or the region where this event is being processed?

I don't think it matters since bucket names are unique, but just curious...

As background, CloudTrail can be set up to receive events from multiple regions. And s3 buckets, even though their names are global, are located in specific regions.

Reply:
message.region identifies the AWS region where the event originated
I think message.detail.awsRegion also identifies the AWS region where the event originated
https://docs.aws.amazon.com/AmazonS3/latest/userguide/notification-content-structure.html
https://docs.aws.amazon.com/AmazonCloudWatch/latest/events/CloudWatchEventsandEventPatterns.html
I am changing the message to reflect this